packages/datagateway-common/src/app.types.tsx

```diff
@@ -83,6 +83,11 @@ export interface User {
   fullName?: string;
 }
 
+export interface FormattedUser {
+  role: string;
+  fullName: string;
+}
```

Comment:
Actually one more thing I forgot - what is the purpose of this? It looks like anywhere that FormattedUser is used, it's already defined as an interface in that file. For example, isisStudyLanding component already has this interface defined locally. Is this being used anywhere? A quick look through your commits indicates this might be leftover code.

Reply:
@sam-glendenning this was actually something I had overlooked, I wanted to move the FormattedUser out into datagateway-common since I was planning on using the type definition in another file. I will delete this, thank you since I did not spot it.